examples/test/test_openstreetmap.js

```diff
@@ -28,13 +27,11 @@ describe("OpenStreetMap example", function() {
             address_list: [
                 "Single Street, Suburb",
             ],
-            key: 'testapikey',
```

Comment:
These changes shouldn't be in your branch -- it looks like develop was merged into this branch badly?